images/multus-daemonset.yml

```diff
@@ -190,6 +190,40 @@ spec:
           mountPath: /host/opt/cni/bin
         - name: multus-cfg
           mountPath: /tmp/multus-conf
+      initContainers:
+        - name: install-multus-binary
+          image: ghcr.io/k8snetworkplumbingwg/multus-cni:stable
+          command:
+            - "cp"
```

Comment:
`cp -f` maybe?

Reply:
Pardon my ignorance, but what is the advantage ?